packages/ui/src/components/Button/button.style.ts

```diff
@@ -59,10 +60,24 @@ export const ButtonStyles = styled.div`
     padding: 0px 10px;
   }
   .medium {
-    height: 32px;
-    padding: 6px 12px;
     min-width: 70px;
+    height: 32px;
     line-height: 0;
+    padding: 6px 12px;
     font-size: 0.875rem;
   }
+  .large {
+    height: 50px;
+    span {
+      position: absolute;
+      display: flex;
+      -webkit-box-align: center;
+      align-items: center;
+      inset: 0px auto 0px 22px;
+      z-index: 1;
+    }
```

Comment:
I would expect we will use the button with an icon even when the size is small. Mind handling this case as well?